library/psa_crypto.c

```diff
@@ -5997,24 +5997,21 @@ psa_status_t mbedtls_psa_inject_entropy( const uint8_t *seed,
 }
 #endif /* MBEDTLS_PSA_INJECT_ENTROPY */
 
-/** Get the key buffer size for the key material in export format
+/** Validate the key type and size for a key generation
```

Comment:
"Validate the key type and size for key generation", or "Validate the key type and size for a key generation operation" are possibly slightly better.

Reply:
Thanks. As I have to rebase I will do the change.

Reply:
Done